axiom/nr_errors.c

```diff
@@ -14,37 +14,68 @@
 #include "util_strings.h"
 #include "util_time.h"
 
-nr_error_t* nr_error_create(int priority,
+static nr_error_t* nr_error_create_helper(int priority,
                             const char* message,
                             const char* klass,
+                            const char* error_file,
+                            int error_line,
+                            const char* error_context,
+                            int error_no,
                             const char* stacktrace_json,
                             const char* span_id,
                             nrtime_t when) {
   nr_error_t* error;
 
-  if (0 == message) {
```

Comment:
Why were these guards removed?

Reply:
These were removed because these guards are being checked before this helper function is called. An example of this is in [nr_error_create](https://github.com/newrelic/newrelic-php-agent/pull/942/files#diff-6f14a71dbed1d482f4f6006b000174fbd0e41f5138cdb61664ea50e9a1aa7947R54).